addons/l10n_rs/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_rs_is_serbian_company = fields.Char(string='Is Serbian Company', compute='_compute_l10n_rs_is_serbian_company')
+    l10n_rs_company_registry = fields.Char(string='Company ID', help='The registry number of the company.')
```

Comment:
Is this really different than the generic company registry? How so?

Reply:
It is the company registry of the customer :wink:

Reply:
Mmmh ok...
Look at this for 15.5 then https://github.com/odoo/odoo/commit/c4bfaf3336a30eda4d12399cd93ee2ffd200d65a

Reply:
Indeed, CHKL told me about that recent feature :ok_hand:
I plan on removing `l10n_rs_company_registry` in `15.5` (with a migration script to transfer the data to `partner.company_registry`).